### DISCUSSION TITLE
Welcome to open-audio Discussions!

### DISCUSSION THREAD

**Discussion by Justmalhar** [Justmalhar/open-audio#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GTech2090** [comment]
Is it possible to change the maximum number of characters in the text field?

**Reply by Justmalhar** [reply]
It is capped by OpenAI currently to 4096 characters. I will update it as a configuration once OpenAI updates it
